src/dotnet/APIView/APIViewWeb/Client/src/reviews.ts

```diff
@@ -1,21 +1,28 @@
 ﻿$(() => {
   const searchBox = $("#searchBox");
 
+  // clear the filter so we don't have to deal with the case of a user navigating backwards and
```

Comment:
Why don't we just run filter function again when the user navigates backwards?

Reply:
We can but then the page jumps after it is initially rendered.

Reply:
meh, it's okiosh